### PR TITLE
[ADP-3093] Temporarily reenable macOS tests

### PR DESCRIPTION
**Pull request by HeinrichApfelmus**
This pull request temporarily re-enables our macOS tests.

### CODE REVIEW

**Review by paolino — APPROVED** [review]
Secret agents ?

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ steps:
     env:
       TMPDIR: "/cache"
 
-  - label: 'Build bemchmarks (linux)'
```
Comment (Unisay) [inline .buildkite/pipeline.yml]: I've seen memchmarks already, bemchmarks I haven't seen yet.

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -144,15 +144,15 @@ steps:
       - './nix/regenerate.sh'
     agents:
       system: ${macos}
-    if: 0 == 1 # Disabled for now until a macos mini is available
+    # if: 0 == 1 # Disabled for now until a macos mini is available
```
Comment (Unisay) [inline .buildkite/pipeline.yml:149]: Any reason to leave it commented out and not deleting?

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (HeinrichApfelmus) [reply .buildkite/pipeline.yml]: Not my doing, but I have fixed it. 😂

File: .buildkite/pipeline.yml (hunk 2 above)
Comment (HeinrichApfelmus) [reply .buildkite/pipeline.yml:149]: Yes. — It's temporary until will get dedicated macOS Agents.

Comment (Unisay) [reply .buildkite/pipeline.yml:149]: Do you mean that once we have those agents you'll un-comment the `if: 0 == 1`  back?

Comment (HeinrichApfelmus) [reply .buildkite/pipeline.yml:149]: > Do you mean that once we have those agents you'll un-comment the if: 0 == 1 back?

No, the other way round — once we have those Agents, I will delete the line.

**Comment by HeinrichApfelmus** [comment]
Closing in favor of https://github.com/cardano-foundation/cardano-wallet/pull/4258 .

**Pull request closed without merging**